hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/VolumeUsage.java

```diff
@@ -69,13 +69,11 @@ SpaceUsageSource realUsage() {
    * <pre>
    * {@code
    * Calculate available space use method B.
-   * |----used----|   (avail)   |++++++++reserved++++++++|
-   *              |     fsAvail      |-------other-------|
-   *                          -&gt;|~~~~|&lt;-
-   *                      remainingReserved
+   * |----used----|          fsAvail           |---other---|
+   * |----used----|   (avail)   |+++reserved+++|---other---|
    * }
    * </pre>
-   * B) avail = fsAvail - Max(reserved - other, 0);
+   * B) avail = fsAvail - Max(reserved, 0);
```

Comment:
@symious , it's a good idea.   The change looks good to me.  Is there any chance that "reserved" is a negative value?  If not, then here we can just use "fsAvail - reserved".

Reply:
Updated, PTAL.